src/main/java/org/broadinstitute/hellbender/utils/dataflow/BucketUtils.java

```diff
@@ -41,4 +41,20 @@ public static InputStream openFile(String path, PipelineOptions popts) {
             throw new UserException.CouldNotReadInputFile(path, x);
         }
     }
+
+    /**
+     * Open a binary file for writing regardless of whether it's on GCS or local disk.
+     * For writing to GCS it'll use the application/octet-stream MIME type.
+     */
+    public static OutputStream createFile(String path, PipelineOptions popts) {
+        try {
+            if (isCloudStorageUrl(path)) {
+                return Channels.newOutputStream(new GcsUtil.GcsUtilFactory().create(popts).create(GcsPath.fromUri(path), "application/octet-stream"));
+            } else {
+                return new FileOutputStream(path);
+            }
+        } catch (Exception x) {
+            throw new GATKException("Couldn't write to "+path);
```

Comment:
Make this a `UserException.CouldNotCreateOutputFile` for consistency with `openFile()` above.